### DISCUSSION TITLE
Fontsource + Storybook

### DISCUSSION THREAD

**Discussion by jacksonblankenship** [fontsource/fontsource#411] [Q&A]
I can't seem to find any documentation about how to integrate fontsource with storybook. Has anybody successfully combined the two?

**Comment by ayuhito** [comment]
I've definitely seen evidence of it being used in Storybook, e.g. this random [PR](https://github.com/openmsupply/openmsupply-client/pull/133) I found. Although I've never really used Storybook and thus can't assist properly. It isn't a dead end to explore though!

**Comment by PauloJorgeS** [comment]
Hello,
I hope it makes sense to add my question here.
I install the @fontsource/inter on my stortybook project.
But when I build my project I don't see the files from `./files/inter-cyrillic-ext-400-normal.woff2'` on my build.

And whoever uses my components gets a file not found.

I'm not getting why my build does not include a folder with all the files that file `400.css` use.
